packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -275,11 +289,22 @@ export abstract class QueueProcessingServiceBase extends Construct {
    * The scaling interval for autoscaling based off an SQS Queue size.
    */
   public readonly scalingSteps: ScalingInterval[];
+
   /**
    * The AwsLogDriver to use for logging if logging is enabled.
    */
   public readonly logDriver?: LogDriver;
 
+  /**
+   * Flag to disable CPU based auto scaling strategy on the service.
+   */
+  public readonly disableCpuBasedScaling: boolean;
+
+  /**
+   * The target CPU utilization percentage for CPU based scaling strategy when enabled.
+   */
+  public readonly cpuTargetUtilizationPercent: number;
```

Comment:
Is there a use case for these being exposed as public properties on the construct? I can't think of one, and would rather default to not adding it. We can always add it in later.

Reply:
I added this one as we are removing the CPU based scaling, if someone really needs that then they can at least control the target step.

Reply:
Or do you mean on the base class?

Reply:
I mean that these should start as `private`, unless there is a valid use case for referencing the property on a QueueProcessingService.

Reply:
Thanks, fixed.